docs/resources/networkloadbalancer.md

```diff
@@ -15,31 +15,31 @@ Manages a **Network Load Balancer**  on IonosCloud.
 
 ```hcl
 resource "ionoscloud_datacenter" "example" {
-	name                    = "Datacenter Example"
-	location                = "us/las"
-	description             = "Datacenter Description"
-	sec_auth_protection     = false
+  name                    = "Datacenter Example"
+  location                = "us/las"
+  description             = "Datacenter Description"
+  sec_auth_protection     = false
 }
 
 resource "ionoscloud_lan" "example1" {
-    datacenter_id           = ionoscloud_datacenter.example.id
-    public                  = false
-    name                    = "Lan Example 1"
+  datacenter_id           = ionoscloud_datacenter.example.id
+  public                  = false
+  name                    = "Lan Example 1"
 }
 
 resource "ionoscloud_lan" "example2" {
-    datacenter_id           = ionoscloud_datacenter.example.id
-    public                  = false
-    name                    = "Lan Example 2"
+  datacenter_id           = ionoscloud_datacenter.example.id
+  public                  = false
+  name                    = "Lan Example 2"
```

Comment:
It's more difficult here, as the bucket needs to be created and we want to be able to copy-paste examples and they should work. Maybe we can add something separately on how to create an s3 bucket, but it's tricky.

Reply:
@cristiGuranIonos maybe write an example only to be used as a reference (with a bucket that doesn't exist), since it won't work to copy-paste it, it should be a good starting point rather than not having any example.

Reply:
added examples for nic and nlb